unity/CAVE/Assets/W3D_Translator/CLI.cs

```diff
@@ -64,7 +67,7 @@ static void Main()
     static Story LoadStory(string xmlPath) {
         try {
             XmlSerializer serializer = new XmlSerializer(typeof(Story));
-            using (XmlReader reader = XmlReader.Create(xmlPath))
+            using (XmlReader reader = XmlReader.Create(xmlPath)) 
```

Comment:
Maybe grab a C# linter to prevent stuff like this? (this should be very very low priority 😉 )

Reply:
Is there any that you recommend?

Reply:
I used StyleCop for Borton Lab

https://github.com/DotNetAnalyzers/StyleCopAnalyzers